test/docfx.Tests/CompositeCommandTest.cs

```diff
@@ -56,23 +54,37 @@ public void TestCommandFromCSCodeToHtml()
 /// </example>
 public class HelloWorld(){}}
 ";
-            var sourceFile = Path.Combine(_projectFolder, "test.cs");
-            File.WriteAllText(sourceFile, sourceCode);
+            var sourceFile = Path.Combine(_projectFolder, "src", "test.cs");
+            CreateFile(sourceFile, sourceCode, "src");
 
             var docfxJson = $@"{{
 ""metadata"": [
     {{
-        ""src"": ""test.cs"",
-        ""cwd"": ""{_projectFolder.ToNormalizedPath()}"",
-        ""dest"": ""{_outputFolder.ToNormalizedPath()}/api""
+        ""src"": ""src/test.cs"",
+        ""dest"": ""api""
     }}
 ],
 ""build"": {{
     ""content"": {{
-        ""files"": ""api/*.yml"",
-        ""cwd"": ""../{Path.GetFileName(_outputFolder)}""
+        ""files"": ""api/*.yml""
     }},
-    ""dest"": ""{_outputFolder.ToNormalizedPath()}/site""
+    ""dest"": ""{_outputFolder.ToNormalizedPath()}/site"",
+    ""sitemap"":{{
+        ""baseUrl"": ""https://dotnet.github.io/docfx"",
+        ""priority"": 0.1,
+        ""changefreq"": ""monthly"",
+        ""fileOptions"":{{
```

Comment:
What about move to FileMappingItem?

Reply:
use `sitemap` to make the feature options straight forward